client/src/pages/NewReport/NodesSection/NodesSection.tsx

```diff
@@ -0,0 +1,149 @@
+import SectionComponent from 'components/SectionComponent/SectionComponent.tsx';
+import Table, { TableColumn } from 'components/Table/Table.tsx';
+import { useEffect, useState } from 'react';
+import TagButton from 'components/TagButton/TagButton.tsx';
+import SVGIcon from 'components/SVGIcon/SVGIcon.tsx';
+import ActionButton, { ActionButtonColor } from 'components/ActionButton/ActionButton.tsx';
+import OverlayComponent from 'components/OverlayComponent/OverlayComponent.tsx';
+import LinkComponent from 'components/LinkComponent/LinkComponent.tsx';
+import CustomPrompt from 'components/CustomPrompt/CustomPrompt.tsx';
+import { ManagmentServiceApiInstance } from 'api/managment-service';
+
+export interface NodeEntry {
+    id: string;
```

Comment:
Same here, make sure that this object matches the contract (or at least allows to source these fields)